snipsnapdev/snipsnap
# feat: add button for create new group in select

**Describe what changes this pull request brings**

This pull request brings button for creating new group on group selector.

**Describe any additional changes**

Move folder `/create-template-group-modal` from `templates/frontend/src/components/shared/sidebar` to `templates/frontend/src/components/shared/`

**Steps to test**
1. Open page `/create-template`
2. Click on `Select group` and then on `Create group`
3. Tape name of the new group
4. Confirm that new group was created

**Screenshots**

<img width="1440" alt="Снимок экрана 2021-07-20 в 21 11 33" src="https://user-images.githubusercontent.com/18118729/126374301-17e8a7a7-3c89-4367-8f67-3764c2e73ef5.png">

**Checklist**

N/A

**TODO**

N/A

**References**

[Issue #96](https://github.com/snipsnapdev/snipsnap/issues/96).
